Source/Models/Scheduler.swift

```diff
@@ -1,9 +1,9 @@
-public typealias Scheduler = ((() -> Void) -> Void)
+public typealias Scheduler = ((@escaping () -> Void) -> Void)
```

Comment:
My bad, the closure may be scheduled asynchronously, so it makes sense for it to be `@escaping` here.